kubejs/server_scripts/constants/metal_properties.js

```diff
@@ -161,33 +161,33 @@ const metal_properties = {
             }
         }
     },
-    elementium: {
-        meltingPoint: 875
-    },
-    enderium: {
-        meltingPoint: 1450
-    },
-    signalum: {
-        meltingPoint: 1272
-    },
-    lumium: {
-        meltingPoint: 1354
-    },
-    manasteel: {
-        meltingPoint: 900
-    },
-    terrasteel: {
-        meltingPoint: 1350
-    },
-    gaia_spirit: {
-        meltingPoint: 975
-    },
+    // elementium: {
+    //     meltingPoint: 875
+    // },
+    // enderium: {
+    //     meltingPoint: 1450
+    // },
+    // signalum: {
+    //     meltingPoint: 1272
+    // },
+    // lumium: {
+    //     meltingPoint: 1354
+    // },
+    // manasteel: {
+    //     meltingPoint: 900
+    // },
+    // terrasteel: {
+    //     meltingPoint: 1350
+    // },
+    // gaia_spirit: {
+    //     meltingPoint: 975
+    // },
     infused_iron: {
         meltingPoint: 850
     },
-    sky_metal: {
-        meltingPoint: 950
-    },
+    // sky_metal: {
+    //     meltingPoint: 950
```

Comment:
Sky metal is from Nature's Aura. Why disable it?

Reply:
Woops, my bad